keps/sig-node/2727-grpc-probe/README.md

```diff
@@ -151,6 +146,18 @@ move users away from using the (portNum, portName) union type.
 - Unit test: Add unit tests to `pkg/kubelet/prober/...`
 - e2e: Add test case and conformance test to `e2e/common/node/container_probe.go`.
 
+### Alternative Considerations
```

Comment:
To avoid confusing this with the “Alternatives” section later on, I recommend either moving this there or leaving it here, and in either case change the section heading to “Naming Considerations”.